### PR TITLE
removed old github workflows and add correct pr build

### PR DESCRIPTION
**Pull request by shokakucarrier**

### CODE REVIEW

**Review by ligangty — COMMENTED** [review]
Seems the action build failed.

File: .github/workflows/maven-build.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+              "id": "sonatype-nexus-snapshots",
+              "username": "${{ secrets.SONATYPE_BOT_USERNAME }}",
+              "password": "${{ secrets.SONATYPE_BOT_TOKEN }}"
+          }]
+
+    - name: Build the Maven verify phase
+      run: mvn -B -V clean verify -Prun-its -Pci
+
+    - name: Deploy the artifact
+      run: mvn help:effective-settings -B -V clean deploy -e
```
Comment (ligangty) [inline .github/workflows/maven-build.yml:44]: And I'm a little confused here.  Will this happen when pr created but not merged in this action?

Comment (shokakucarrier) [reply .github/workflows/maven-build.yml:44]: it will, I'll make a change on that

**Comment by shokakucarrier** [comment]
> Seems the action build failed.

It's because I never contributed on this repo, so organization secret is not avaliable

**Comment by shokakucarrier** [comment]
@ligangty now this is working correctly

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**